nerfstudio/models/nerfplayer_nerfacto.py

```diff
@@ -119,7 +119,8 @@ def populate_modules(self):
         self.density_fns = []
         num_prop_nets = self.config.num_proposal_iterations
         # Build the proposal network(s)
-        self.proposal_networks = torch.nn.ModuleList()
+        # The following line uses a typing workaround (may not be the best way to do this)
+        self.proposal_networks = cast(List[TemporalHashMLPDensityField], torch.nn.ModuleList())
```

Comment:
This line is not good. Any suggestions?

Reply:
The best I can think of is a `PROPOSAL_NETWORKS_TYPE = List[TemporalHashMLPDensityField]` line at the top and then just do a `self.proposal_networks = cast(PROPOSAL_NETWORKS_TYPE, torch.nn.ModuleList())`

Reply:
I'm okay with what you originally had, but I pushed an alternative solution where we instead cast the `ModuleList` instance to `Sequence[TemporalHashMLPDensityField]`. This seems a little bit more "correct" since `Sequence` is a protocol, although in practice either seem OK?